src/validators/common/mongodb.validators.js

```diff
@@ -0,0 +1,23 @@
+import { body, param } from "express-validator";
+
+/**
+ *
+ * @param {string} idName
+ * @returns ValidationChain[]
+ * @description A common validator responsible to validate mongodb ids passed in the url's path variable
+ */
+export const mongoIdPathVariableValidator = (idName) => {
+  return [
+    param(idName).notEmpty().isMongoId().withMessage(`Invalid ${idName}`),
+  ];
+};
+
+/**
+ *
+ * @param {string} idName
+ * @returns ValidationChain[]
```

Comment:
remove the `@returns ValidationChain[]` tag. Javascript is automatically inferring the return type.